app/src/main/res/layout/tab_information_fragment.xml

```diff
@@ -166,7 +160,7 @@
                         <com.github.mikephil.charting.charts.PieChart
                             android:id="@+id/piechartContainers"
                             android:layout_width="match_parent"
-                            android:layout_height="200dp" />
+                            android:layout_height="200dp"/>
```

Comment:
The labels in the Container chart are not displayed.
You changed the margin of the whole CardLayout but you haven't changed the height of the layout.

Make sure to display the labels as well